### PR TITLE
Soeren/4 spaces indentation in style css

### PR DESCRIPTION
**Pull request by sisou**
#168 changed the indentation to 2 spaces for whatever reason.

I like 4 spaces more, like it was before (uh, that rhymes :D)

(already includes title font fix from #193)

### CODE REVIEW

**Comment by paberr** [comment]
Personally, I like 4 spaces more as well.
Since this PR is not urgent, I'll take the time and ask the others for their opinion as well.

File: clients/browser/style.css [hunk 1]
```diff
@@ -1,392 +1,392 @@
...
+    background: rgba(255, 255, 255, 0.42);
+    color: #FFF;
+    border-top-left-radius: 3px;
+    border-top-right-radius: 3px;
 }
 
 .info-block .icon-nimiq::before {
-    font-size: 12px;
-    margin-right: 2px;
+        font-size: 12px;
```
Comment (styppo) [inline clients/browser/style.css]: Double indent

File: clients/browser/style.css [hunk 2]
```diff
@@ -1,392 +1,392 @@
...
+        margin-right: 2px;
```
Comment (styppo) [inline clients/browser/style.css]: Double indent

File: clients/browser/style.css [hunk 3]
```diff
@@ -1,218 +1,218 @@
...
-  float: left;
-  width: 50%;
+    float: left;
+    width: 50%;
 }
 
 @media screen and (max-width: 760px) {
-  .left, .right {
+    .left, .right {
     width: 100%;
```
Comment (styppo) [inline clients/browser/style.css]: Missing indent

Comment (styppo) [inline clients/browser/style.css]: Missing indent

**Pull request merged**